parkingspaces/src/main/java/com/baspiotr/parkingspaces/domain/services/ParkingService.java

```diff
@@ -0,0 +1,47 @@
+package com.baspiotr.parkingspaces.domain.services;
+
+import com.baspiotr.parkingspaces.domain.model.Parking;
+import com.baspiotr.parkingspaces.domain.model.User;
+import com.baspiotr.parkingspaces.domain.repository.ParkingRepository;
+import com.baspiotr.parkingspaces.domain.repository.UserRepository;
+import com.baspiotr.parkingspaces.domain.utils.Fee;
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
+public class ParkingService {
+
+    private final ParkingRepository parkingRepository;
+    private final UserRepository userRepository;
+
+    //todo 2) As a parking operator, I want to check if the vehicle has started the parking meter
+
+    public boolean isCarStartedParkingMeterByUserId(int userId) {
+
+        User user = userRepository.findOne(userId);
```

Comment:
what if there is no such user ?
you would need to catch nulls and throw runtime exceptions
https://spring.io/blog/2013/11/01/exception-handling-in-spring-mvc